### ISSUE TITLE
Ideate and crafting the master website content publishing Git template

### ISSUE THREAD

**Issue by JennySanchez** [carbon-design-system/digital-design-ideation#178]
## We need a master publishing template for Roberta to create website content publishing task issues for DDS team members.

**Comment by JennySanchez** [comment]
## Objective
- Provide Carbon for IBM.com adopter the best practices and guidance they need to adopt Carbon for IBM.com.
- Present Carbon for IBM.com adopters the resources and tools we offer as soon as it's available.

## General publishing process
Please make sure you always start the content crafting process by viewing the **[2020 website content publishing process box document](https://ibm.box.com/s/6oiqsvhxtchi1lfozyuw1soje67unbmj)** first.


 ## General publishing guidelines

#### Publishing for Components section website page
- Content crafting starts as soon as the design is finalized. 
- Components section includes UI components and Layout components.
- The content can be at the MVP level. MVP means the content meets the basic needs of an adopter for learning. It does not mean the content crafting will or should stop at this stage. Future enhancement should be working on with a separate task issue.
- MVP content publishing template and requirement can be found in the Publishing template and best practices/[_UI and Layout Components folder](https://ibm.box.com/s/8p9k496ohera4aa9zge9syxyehz5p7h0).
- Take a look at the [storybook](https://ibmdotcom-react.mybluemix.net/?path=/story/overview-getting-started--page) - so you know how the component works. This will also be a reference to accurately set up the images for the website page. Reference image guidelines can be found in the Publishing template and best practices folder. 
- If Storybook is unavailable, please use the visual and design spec files inside of the [Carbon for IBM.com Box folder](https://ibm.box.com/s/vo3u9mxn8y7a5d7zoqvrvz7aalbf2yu3) as a reference. 
- Content publishing to the live website can be code or markdowns. If you need guidance on how to start with markdowns, please see the step by step instruction in the [Editing and publishing in Github Box document](https://ibm.box.com/s/as8fo8gcecvizngxgq2dpuk4po2alq7t).
- When the page is published, please also make sure the components table list on the [Overview page](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/overview) is updated as part of the publishing task.

#### Publishing for Patterns section website page
- Content crafting starts as soon as Initial research and publishing agreements are accepted by the DDS leadership team.
- The content can be at the MVP level. MVP means the content meets the basic needs of an adopter for learning. It does not mean the content crafting will or should stop at this stage. Future enhancement should be working on with a separate task issue.
- MVP content publishing template and requirement can be found in the Publishing template and best practices/[_General page folder](https://ibm.box.com/s/ts8m1a3bb5z5vwexrmh0c1ztthdm8nej).
- Content publishing to the live website can be code or markdowns. If you need guidance on how to start with markdowns, please see the step by step instruction in the [Editing and publishing in Github Box document](https://ibm.box.com/s/as8fo8gcecvizngxgq2dpuk4po2alq7t).
- When the page is published, please also make sure the patterns table list on the [Overview page](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/patterns) is updated as part of the publishing task.

#### Publishing for Guidelines section website page
- Content crafting starts as soon as Initial research and publishing agreements are accepted by the DDS leadership team.
- The content can be at the MVP level. MVP means the content meets the basic needs of an adopter for learning. It does not mean the content crafting will or should stop at this stage. Future enhancement should be working on with a separate task issue.
- MVP content publishing template and requirement can be found in the Publishing template and best practices/[_General page folder](https://ibm.box.com/s/ts8m1a3bb5z5vwexrmh0c1ztthdm8nej).
- Content publishing to the live website can be code or markdowns. If you need guidance on how to start with markdowns, please see the step by step instruction in the [Editing and publishing in Github Box document](https://ibm.box.com/s/as8fo8gcecvizngxgq2dpuk4po2alq7t).
- When the page is published, please also make sure the guideline table list on the [Overview page](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/guidelines) is updated as part of the publishing task.


#### Publishing for other website pages
- Content crafting starts as soon as Initial research and publishing agreements are accepted by the DDS leadership team.
- The content can be at the MVP level. MVP means the content meets the basic needs of an adopter for learning. It does not mean the content crafting will or should stop at this stage. Future enhancement should be working on with a separate task issue.
- MVP content publishing template and requirement can be found in the Publishing template and best practices/[_General page folder](https://ibm.box.com/s/ts8m1a3bb5z5vwexrmh0c1ztthdm8nej).
- If you do not see the General page folder templates and guidelines meet your need, please reach out to Jenny for further guidance.
- Content publishing to the live website can be code or markdowns. If you need guidance on how to start with markdowns, please see the step by step instruction in the [Editing and publishing in Github Box document](https://ibm.box.com/s/as8fo8gcecvizngxgq2dpuk4po2alq7t).
- Once the Pull Request is submitted and the Preview link is ready, include Jenny, Linda, Jeff, and Wonil as PR Reviewers and Approvers. When there are no additional edits needed. Jenny will merge the PR.

## File naming and storage
- Name the Sketch file with the following convention: Name of the category + components/pattern/guideline + version. ex: Components-Feature card block v1.0, Guidelines-Expressive theme v1.0, Help-Overview v1.0 and etc.

- Please save all content publishing work files (box, markdowns link, or Sketch file) in the proper sub-folder under the[ ✍️ Content + Design folder](https://ibm.box.com/s/tty3jhshkt6l18jkn2v5ac2uznyjexns).

## Publishing acceptance criteria
- [ ] Content crafting is done.
- [ ] Asset image creation is done.
- [ ] Page design is done.
- [ ] Publishing with markdowns is done.
- [ ] QA'd by Jenny.
- [ ] Web page preview is approved by Linda, Jeff, and Wonil.
- [ ] PR merged by Jenny.
- [ ] Web page is published and live on the website.

**Comment by JennySanchez** [comment]
Based on the discussion with Roberta today, we will create individual task templates on the website repo as the next step.

**Comment by JennySanchez** [comment]
All four content crafting and publishing Git templates and one content update Git template are created and located in the "Icebox" column on the website repo.

![Screen Shot 2020-11-06 at 5 36 01 PM](https://user-images.githubusercontent.com/45213839/98424196-9ab8a300-2056-11eb-9680-0572471b7e46.png)
